### PR TITLE
Patch test databases [release/97]

### PR DESCRIPTION
**Pull request by mkszuba**
As the change set demonstrates, the last test-DB patching in this repository had taken place before all the necessary schema patches got published.

### CODE REVIEW

File: modules/t/test-genome-DBs/ontology/ontology/table.sql [hunk 1]
```diff
@@ -197,13 +197,13 @@ CREATE TABLE `synonym` (
 CREATE TABLE `term` (
   `term_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
   `ontology_id` int(10) unsigned NOT NULL,
-  `subsets` text CHARACTER SET latin1,
+  `subsets` text CHARACTER SET latin1 DEFAULT NULL,
   `accession` varchar(64) CHARACTER SET latin1 NOT NULL,
   `name` varchar(255) CHARACTER SET latin1 NOT NULL,
-  `definition` text CHARACTER SET latin1,
+  `definition` text CHARACTER SET latin1 DEFAULT NULL,
   `is_root` int(11) DEFAULT NULL,
```
Comment (ens-bwalts) [inline modules/t/test-genome-DBs/ontology/ontology/table.sql]: These don't match is_root and is_obsolete in the patches for ensembl-rest and ensembl (NULL vs 0). Is this OK?

Comment (mkszuba) [reply modules/t/test-genome-DBs/ontology/ontology/table.sql]: No, it is not. The interesting thing is, either I am blind or there is no patch for this in _ols-ensembl-loader_ - yet most repos do use 0 here.

Comment (marcoooo) [reply modules/t/test-genome-DBs/ontology/ontology/table.sql]: Actually, the field is now a : `definition` text COLLATE utf8_general_ci. Does it cause a problem on your side?

Comment (mkszuba) [reply modules/t/test-genome-DBs/ontology/ontology/table.sql]: Have just discussed this with @marcoooo and the _ensembl-io_ test ontology database is seriously out of date for some reason, in spite of the fact the meta table lists all patches as applied. We'll simply sync it with the one from _ensembl_, which appears to be up to date.

**Pull request merged**